helper/Config.php

```diff
@@ -40,7 +40,10 @@ public function isValidNamespace(string $namespace): bool
         }
 
         $namespaces = explode(',', $this->namespaces);
-
+        // Handle root namespace
+        if ($namespace === '') {
+            return in_array(':', $namespaces, true);
```

Comment:
why not fix the other code to return `:` for root namespace? so change here would not be neccessary.

Reply:
It seems to work - at least the modified version for discord does. You're welcome to change it if you like.

Reply:
can you make the change? this change is not related to what service is notified but internal code how data is passed around.

Reply:
After 4 months, I don't remember what the code does anymore. Plus I don't even use this plugin; I use the discordnotifier fork. I simply applied that same edits here as I did there. In other words, I have no way of testing any further changes made

Reply:
- [`fd7d520` (#30)](https://github.com/glensc/dokuwiki-plugin-slacknotifier/pull/30/commits/fd7d520c35098530455d3a0600d0372cf293bb5c)
- [`1146441` (#30)](https://github.com/glensc/dokuwiki-plugin-slacknotifier/pull/30/commits/114644185c90de17dfd8028d73cf43eddb7d6fdd)

Reply:
I can't test this either right now. I can't promise when I can test this myself.